python/whylogs/api/pyspark/experimental/__init__.py

```diff
@@ -1,5 +1,6 @@
 from .profiler import (
     collect_column_profile_views,
+    collect_dataset_profile_view,
```

Comment:
Good for now, but I'd like to think through the API names here. They are a bit long for the API although they are accurate. something like DatasetCollecter or something like that may be a easier to use, but still accurate?

Reply:
I'm with you on these being names we likely change, which is partly why these are in "experimental" namespace. I'm not against longer explicit naming for advanced scenarios in particular. Maybe we can get some input from users in our RC on this?